dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -1277,14 +1277,43 @@ public static boolean isFieldEqualToEnrolledAt( String fieldName )
             return ENROLLED_AT.getPropName().equalsIgnoreCase( fieldName );
         }
 
+        /**
+         * Gets the column in this enum that matches the property
+         *
+         * @param property to match with a column
+         * @return a column matching the provided field or an exception if no
+         *         column is found
+         */
         public static String getColumn( String property )
         {
-            return Arrays.stream( values() )
+            return getColumn( property, "" );
+        }
+
+        /**
+         * Gets the column in this enum that matches the property and
+         * substitutes the default alias with the one given as a parameter
+         *
+         * @param property to match with a column
+         * @param alias to replace the default with, if empty, the default will
+         *        be kept
+         * @return a column with the provided alias or an exception if no column
+         *         is found
+         */
+        public static String getColumn( String property, String alias )
+        {
+            String column = Arrays.stream( values() )
                 .filter( orderColumn -> orderColumn.getPropName().equals( property ) )
                 .map( OrderColumn::getColumn )
                 .findFirst()
                 .orElseThrow(
                     () -> new IllegalArgumentException( "Property" + property + " is not defined as order column" ) );
+
```

Comment:
The getColumn method should just return a value and not an exception. If we want to add an exception we should add a validation when we collect the order values from a request. At this level, I think we should log